pkg/tenant/db/errors.go

```diff
@@ -3,16 +3,17 @@ package db
 import "errors"
 
 var (
-	ErrNotConnected         = errors.New("not connected to trtl database")
-	ErrNotFound             = errors.New("object not found for the specified key")
-	ErrUnavailable          = errors.New("trtl database service is unavailable")
-	ErrMissingID            = errors.New("object requires id for serialization")
-	ErrMissingOrgID         = errors.New("object requires organization id for serialization")
-	ErrMissingTenantID      = errors.New("object requires tenant id for serialization")
-	ErrMissingProjectID     = errors.New("object requires project id for serialization")
-	ErrNumberFirstCharacter = errors.New("name cannot begin with a number")
-	ErrSpecialCharacters    = errors.New("name cannot include special characters")
-	ErrMissingMemberName    = errors.New("member name is required")
-	ErrMissingProjectName   = errors.New("project name is required")
-	ErrMissingTopicName     = errors.New("topic name is required")
+	ErrNotConnected       = errors.New("not connected to trtl database")
+	ErrNotFound           = errors.New("object not found for the specified key")
+	ErrUnavailable        = errors.New("trtl database service is unavailable")
+	ErrMissingID          = errors.New("object requires id for serialization")
+	ErrMissingOrgID       = errors.New("object requires organization id for serialization")
+	ErrMissingTenantID    = errors.New("object requires tenant id for serialization")
+	ErrMissingProjectID   = errors.New("object requires project id for serialization")
+	ErrValidation         = errors.New("name cannot begin with a number or include a special character")
```

Comment:
Can we update the name of this error to mention the name somehow? e.g. `ErrNameValidation`

Reply:
I added the following, so that the correct name will appear. 

> func ValidatonError(model string) error {
	return fmt.Errorf("%s name cannot begin with a number or include a special character", model)
}